packages/json-rpc-middleware-stream/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@metamask/json-rpc-middleware-stream",
+  "version": "6.0.0",
+  "description": "A small toolset for streaming JSON-RPC data and matching requests and responses",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/json-rpc-middleware-stream#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "ISC",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build:docs": "typedoc",
+    "changelog:validate": "../../scripts/validate-changelog.sh @metamask/json-rpc-middleware-stream --tag-prefix-before-package-rename json-rpc-middleware-stream@ --version-before-package-rename 5.0.1",
+    "publish:preview": "yarn npm publish --tag preview",
+    "test": "jest",
+    "test:clean": "jest --clearCache",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@metamask/json-rpc-engine": "^7.1.1",
+    "@metamask/safe-event-emitter": "^3.0.0",
+    "@metamask/utils": "^8.1.0",
+    "readable-stream": "^3.6.2"
+  },
+  "devDependencies": {
+    "@metamask/auto-changelog": "^3.4.2",
+    "@types/jest": "^27.4.1",
+    "@types/readable-stream": "^2.3.0",
```

Comment:
Nit: It looks like this is ^2.3.9 in the [original repo](https://github.com/MetaMask/json-rpc-middleware-stream/blob/main/package.json):

``` suggestion
    "@types/readable-stream": "^2.3.9",

Reply:
Yes this is enforced by yarn constraints. Setting this to "^2.3.9" triggers an error because the least (and only) version used in the monorepo is "^2.3.0". 

If using the latest version for a certain dependency is important, we need to open a PR in core to bump it throughout the monorepo first. Otherwise, this needs to be decremented for now.

Reply:
Oh, I swear I searched the repo for this package. I see now it's used by `accounts-controller`. Okay, looks good then!